### PR TITLE
docs: Improved Opening Balance Documentation

### PR DESCRIPTION
**Pull request by asbasawaraj**

### CODE REVIEW

File: erpnext_com/www/docs/user/manual/en/accounts/opening-balance.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+**The opening balance is the balance that is brought forward at the beginning of an accounting period from the end of a previous accounting period or when starting out.**
+
+## 1. Introduction
+
+If you are a new company you will have minimal opening balances to be imported. However, if you are migrating from a legacy accounting system like Tally or a Fox Pro based software you will have considerable data to be imported as opening balance.
+
+We recommend that you start using ERPNext for accounting from a new financial year, but you could start midway too. To setup your accounts, you will need the following for the “day” you start using accounting in ERPNext:
+
+##### Assets:
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/opening-balance.md]: do a pull on frappe_theme and see if the heading levels seem ok, the sizes were changed
adding numbers to sub headings would look nice, 1.1, 1.2....

File: erpnext_com/www/docs/user/manual/en/accounts/opening-balance.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+1. Accounts Payables(AP) i.e. unpaid invoices which your suppliers have sent you
+
+
+If you were using another accounting software before, you should close financial statements in that software first. The closing balance of the accounts should be updated as an opening balance in the ERPNext. Before starting to update opening balance, ensure that your [Chart of Accounts](/docs/user/manual/en/accounts/chart-of-accounts) has all the Accounts required.
+
+> Opening entry is only for Balance Sheet accounts and not for Profit and Loss Accounts.
+
+## 2. Opening Balance of Assets
+
+2.1 [Fixed Assets](/docs/user/manual/en/accounts/opening-balance/fixed_assets)
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/opening-balance.md]: so these could be regular number bullets

File: erpnext_com/www/docs/user/manual/en/accounts/opening-balance.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+3.3 [Accounts Payable](/docs/user/manual/en/accounts/opening-balance/accounts_payable)
+
+## 4. Verify the Opening Balance
+
+Once all assets and liabilities have been imported, the balance of **Temporary Opening** ledger should be zero.
+
+After completing the accounting entries, the trial balance report will look as shown below.
+
+<img class="screenshot" alt="Opening Account" src="{{docs_base_url}}/assets/img/accounts/opening-4.png">
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/opening-balance.md]: image with this name not present in PR

File: erpnext_com/www/docs/user/manual/en/accounts/opening-balance/accounts_payable.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+<!-- add-breadcrumbs -->
+# Import Accounts Payables
+
+**Accounts Payables is the balance amount you owe your suppliers towards the invoices they have already sent you.**
+
+You can import the accounts payables details using Opening Invoice Creation Tool.
+
+To import outstanding invoices:
+
+1. Go to **Accounting > Masters and Accounts > opening Invoice Creation Tool.**
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/opening-balance/accounts_payable.md]: Opening, caps O

File: erpnext_com/www/docs/user/manual/en/accounts/opening-balance/accounts_payable.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/opening-balance/accounts_payable.md]: no need for spaces between the bullets, compact looks alright

File: erpnext_com/www/docs/user/manual/en/accounts/opening-balance/accounts_payable.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+<div>
+    <div class="embed-container">
+        <iframe src="https://www.youtube.com/embed/vfWmugaO1zw" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen>
+        </iframe>
+    </div>
+</div>
+
+Purchase Invoices will be created. You can post Payment Entry against these invoices as and when you make the payment.
+
+<next>
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/opening-balance/accounts_payable.md]: {next} ...brackets!

File: erpnext_com/www/docs/user/manual/en/accounts/opening-balance/accounts_receivable.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+# Import Accounts Receivables
+
+**Accounts Receivables is the balance amount your customer owes you towards the sales invoices you have already sent to them.**
+
+You can import the accounts receivables details using Opening Invoice Creation Tool.
+
+To import outstanding invoices:
+
+1. Go to **Accounting > Masters and Accounts > opening Invoice Creation Tool.**
+
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/opening-balance/accounts_receivable.md]: extra line not needed

File: erpnext_com/www/docs/user/manual/en/accounts/opening-balance/capital_accounts.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
+<!-- add-breadcrumbs -->
+# Import Capital Accounts
+
+Before importing capital accounts, make sure that you have created the required ledgers under **Chart of Accounts > Liabilities > Capital Accouns** group.
+
+To import details of capital accounts:
+
+1. Go to: **Accounts > Masters and Accounts > Journal Entry > New**.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/opening-balance/capital_accounts.md]: is accounting as per v12

File: erpnext_com/www/docs/user/manual/en/accounts/opening-balance/current_assets.md [hunk 9]
```diff
@@ -0,0 +1,17 @@
+<!-- add-breadcrumbs -->
+# Import Current Assets
+
+Before importing current assets, make sure that you have created the required ledgers under **Chart of Accounts > Assets > Current Assets** group.
+
+To import details of current assets:
+
+1. Go to: **Accounts > Company and Accounts > Journal Entry > New**.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/opening-balance/current_assets.md]: update as per v12

File: erpnext_com/www/docs/user/manual/en/accounts/opening-balance/current_liabilities.md [hunk 10]
```diff
@@ -0,0 +1,17 @@
+<!-- add-breadcrumbs -->
+# Import Current Liabilities
+
+Before importing current liabilities, make sure that you have created the required ledgers under **Chart of Accounts > Liabilities > Current Liabilities** group.
+
+To import details of current liabilities:
+
+1. Go to: **Accounts > Masters and Accounts > Journal Entry > New**.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/opening-balance/current_liabilities.md]: Accounting

File: erpnext_com/www/docs/user/manual/en/accounts/opening-balance/current_liabilities.md [hunk 11]
```diff
@@ -0,0 +1,17 @@
...
+1. Enter Posting Date.
+1. Select the appropriate current liability ledgers in Account column and enter the value in Credit column.
+1. Select 'Temporary Opening' ledger in Account and enter the balancing amount in Debit column.
+1. Set 'Is Opening' to Yes.
+
+<img class="screenshot" alt="Stock Asset Ledger" src="{{docs_base_url}}/assets/img/accounts/opening_balance_current_liabilities.png">
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/opening-balance/current_liabilities.md]: replace image, top left is bad

File: erpnext_com/www/docs/user/manual/en/accounts/opening-balance/fixed_assets.md [hunk 12]
```diff
@@ -0,0 +1,37 @@
+<!-- add-breadcrumbs -->
+# Import Fixed Assets
+
+To import all the existing fixed assets first create the asset record and then create a journal entry to update general ledger.
+
+### 1. Create Asset Records
+
+Create Asset record for each asset that your company owns which is not fully depreciated.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/opening-balance/fixed_assets.md]: 'which can be still used'  sounds simpler

File: erpnext_com/www/docs/user/manual/en/accounts/opening-balance/fixed_assets.md [hunk 13]
```diff
@@ -0,0 +1,37 @@
...
+
+1. Go to **Assets > Assets > New.**
+1. Enter Asset Name.
+1. Enter Item Code.
+1. Enter Location.
+1. Enter Purchase Date.
+1. Enter Gross Amount.
+1. Tick **Is Existing Asset.**
+
+<img class="screenshot" alt="Stock Asset Ledger" src="{{docs_base_url}}/assets/img/accounts/asset_opening_balance.png">
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/opening-balance/fixed_assets.md]: this image was probably not needed

File: erpnext_com/www/docs/user/manual/en/accounts/opening-balance/fixed_assets.md [hunk 14]
```diff
@@ -0,0 +1,37 @@
...
+
+
+### 2. Created Journal Entry to Update the Ledgers
+
+When you create an Asset with 'Is Existing Asset' checkbox ticked, the General Ledger is not updated. You will have to update the value via a Journal Entry.
+
+To create a new Journal Entry:
+
+1. Go to: **Accounts > Masters and Accounts > Journal Entry > New.**
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/opening-balance/fixed_assets.md]: Accounting

File: erpnext_com/www/docs/user/manual/en/accounts/opening-balance.md (hunk 3 above)
Comment (asbasawaraj) [reply erpnext_com/www/docs/user/manual/en/accounts/opening-balance.md]: It already exists... not a new screenshot.

https://github.com/frappe/erpnext_com/blob/master/erpnext_com/www/docs/assets/img/accounts/opening-4.png

File: erpnext_com/www/docs/user/manual/en/accounts/opening-balance/fixed_assets.md (hunk 12 above)
Comment (asbasawaraj) [reply erpnext_com/www/docs/user/manual/en/accounts/opening-balance/fixed_assets.md]: No, some assets are still usable but already fully depreciated. And as per my understanding there is no need to bring such assets into the new system.

File: erpnext_com/www/docs/user/manual/en/accounts/opening-balance.md [hunk 15]
```diff
@@ -0,0 +1,69 @@
+<!-- add-breadcrumbs -->
+# Opening Balance in Accounts
+
+**The opening balance is the balance that is brought forward at the beginning of an accounting period from the end of a previous accounting period or when starting out.**
+
+## 1. Introduction
+
+If you are a new company you will have minimal opening balances to be imported. However, if you are migrating from a legacy accounting system like Tally or a Fox Pro based software you will have considerable data to be imported as opening balance.
+
+We recommend that you start using ERPNext for accounting from a new financial year, but you could start midway too. To setup your accounts, you will need the following for the “day” you start using accounting in ERPNext:
```
[the remark this answers is not in the document]
Comment (asbasawaraj) [reply erpnext_com/www/docs/user/manual/en/accounts/opening-balance.md]: its the 'day'

File: erpnext_com/www/docs/user/manual/en/accounts/opening-balance/fixed_assets.md (hunk 13 above)
Comment (asbasawaraj) [reply erpnext_com/www/docs/user/manual/en/accounts/opening-balance/fixed_assets.md]: why not...

Comment (prasadarr) [reply erpnext_com/www/docs/user/manual/en/accounts/opening-balance/fixed_assets.md]: just a general observation, too many images can make the page a bit crowded

File: erpnext_com/www/docs/user/manual/en/accounts/articles/index.md [hunk 16]
```diff
@@ -32,3 +32,6 @@
 ##### 2.5 Reports and Errors
 1. [Purchase Invoice Account Type Error](/docs/user/manual/en/accounts/articles/purchase-invoice-account-type-error)
 1. [Fiscal Year Error](/docs/user/manual/en/accounts/articles/fiscal-year-error)
+
+##### 2.6 Stock and Accounting Integration
+1. [Warehouse and Accounting Ledger Link](/docs/user/manual/en/accounts/articles/warehouse-ledger-link)
```
[the remark this answers is not in the document]
Comment (prasadarr) [reply erpnext_com/www/docs/user/manual/en/accounts/articles/index.md]: pending

**Pull request merged**